pkg/kubelet/metrics/collectors/resource_metrics.go

```diff
@@ -122,7 +146,7 @@ func (rc *resourceMetricsCollector) CollectWithStability(ch chan<- metrics.Metri
 	defer func() {
 		ch <- metrics.NewLazyConstMetric(resourceScrapeResultDesc, metrics.GaugeValue, errorCount)
 	}()
-	statsSummary, err := rc.provider.GetCPUAndMemoryStats(ctx)
+	statsSummary, err := rc.provider.Get(ctx, false)
```

Comment:
Does getting all the metrics increase load within the kubelet/system?

Reply:
Thanks a lot for your feedback!
You're absolutely right. Let me reshape that.

Reply:
Done! :+1: 
Thanks for your feedback